packages/pigeon/platform_tests/ios_unit_tests/ios/Runner/async_handlers.h

```diff
@@ -0,0 +1,27 @@
+// Autogenerated from Pigeon (v0.1.19), do not edit directly.
+// See also: https://pub.dev/packages/pigeon
+#import <Foundation/Foundation.h>
+@protocol FlutterBinaryMessenger;
+@class FlutterError;
+@class FlutterStandardTypedData;
+
+NS_ASSUME_NONNULL_BEGIN
+
+@class Value;
+
+@interface Value : NSObject
+@property(nonatomic, strong, nullable) NSNumber *number;
+@end
+
+@interface Api2Flutter : NSObject
+- (instancetype)initWithBinaryMessenger:(id<FlutterBinaryMessenger>)binaryMessenger;
+- (void)calculate:(Value *)input completion:(void (^)(Value *, NSError *_Nullable))completion;
+@end
+@protocol Api2Host
+- (void)calculate:(Value *_Nullable)input
```

Comment:
`Value *_Nullable` should be `nullable Value` since it's a method parameter, not in the block.

Reply:
done